CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## Unreleased 
+## 20.10.0
 
 ### Features Added
 - Added "eth_signTransaction" JSON RPC
+- Docker namespace updated to reflect "consensys/quorum-ethsigner"
 
```

Comment:
Since the 20th of Aug 0.7.1 release noticed a few other features/bug fixes they might make sense to add change log entries for
* d9c3db97639de0c6007857de7277b9f3fb21f476 metrics endpoint (#328)
* 418b106568ac987af2794253c62c74c3dbac82bb Ensure txns are handled in parallel (#331)
* 76dc4e1df3f60ab5b5726567730bc2e760516840 Remove access-control-allow-origin header from web3provider response